projects/step-core/src/lib/components/plan-tree/plan-tree.component.ts

```diff
@@ -62,30 +62,51 @@ export class PlanTreeComponent implements TreeActionsService {
     { id: PlanTreeAction.MOVE_RIGHT, label: 'Move Right (Ctrl + ➡️)' },
   ];
 
-  getActionsForNode(node: ArtefactTreeNode): Observable<TreeAction[]> {
+  private actionsMultiple: TreeAction[] = [
+      { id: PlanTreeAction.OPEN, label: 'Open (Ctrl + O)', disabled: true },
+      { id: PlanTreeAction.RENAME, label: 'Rename (F2)', disabled: true },
+      { id: PlanTreeAction.ENABLE, label: 'Enable All Selected (Ctrl + E)', hasSeparator: true},
+      { id: PlanTreeAction.DISABLE, label: 'Disable All Selected (Ctrl + E)', hasSeparator: true},
+      { id: PlanTreeAction.COPY, label: 'Copy All Selected (Ctrl + C)'},
+      { id: PlanTreeAction.PASTE, label: 'Paste All Selected (Ctrl + V)'},
```

Comment:
```suggestion
      { id: PlanTreeAction.PASTE, label: 'Paste (Ctrl + V)'},
```
While testing I noticed that "Paste All Selected" doesn't make sense, since we do not paste the selected but the previously copied.